src/controllers/order-controller.js

```diff
@@ -64,8 +65,33 @@ async function createOrder(req, res, next) {
   }
 }
 
+async function updateDeliveryStatus(req, res, next) {
+  const { orderId } = req.params;
+
+  try {
+    const findQuery = { _id: orderId };
+    const order = await OrderModel.findOne(findQuery).select('ownerId');
+
+    if (!order) {
+      throw new NotFoundError('Order not found');
+    }
+
+    if (String(order.ownerId) !== req.userData._id) {
+      throw new ForbiddenError('Not Authorized!');
+    }
+
+    const { deliveryStatus } = req.body;
```

Comment:
always declare body params variables on the top, to see what this controller function does